app/components/embed/media_with_companion_windows_component.html.erb

```diff
@@ -53,9 +77,21 @@
     <div class="right-drawer collapse" data-media-target="rightDrawer">
       <div class="inner">
         <div data-media-target="contents" hidden>
+          <div class="MuiToolbar-root MuiToolbar-regular">
```

Comment:
Can you remove these Mui* classes?  We aren't using them. I think just the h3 will be fine.

Reply:
Good eye, that was indeed cruft.